embrace-android-core/src/test/java/io/embrace/android/embracesdk/internal/envelope/session/SessionPayloadSourceImplTest.kt

```diff
@@ -33,6 +35,7 @@ internal class SessionPayloadSourceImplTest {
         }
         activeSpan = FakePersistableEmbraceSpan.started()
         spanRepository = SpanRepository()
+        spanRepository.trackStartedSpan(checkNotNull(currentSessionSpan.sessionSpan))
```

Comment:
Manually patch through the session span so that it will be included in the final payload that we validate